### ISSUE TITLE
Fatal JavaScript out of memory issue Fedora 39 with XDE Plasma 5 aarch64

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#36681]
Does this issue occur when all extensions are disabled?: Yes

VS Code Version: 1.85.1
Commit: 0ee08df0cf4527e40edc9aa28f4b5bd38bbff2b2
Date: 2023-12-13T09:48:59.576Z
Electron: 25.9.7
ElectronBuildId: 25551756
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Linux arm64 6.6.8-200.fc39.aarch64

Steps to Reproduce:
1. Try and open VS Code. In some instances it will successfully open without the fatal error. However, there are some instances when it will error with the fatally. It doesn't always error.

```
<--- Last few GCs --->

[19188:0x34002e4000]       18 ms: Mark-Compact (reduce) 0.7 (3.0) -> 0.7 (1.8) MB, 1.62 / 0.00 ms  (average mu = 0.023, current mu = 0.016) last resort; GC in old space requested
[19188:0x34002e4000]       19 ms: Mark-Compact (reduce) 0.7 (1.8) -> 0.7 (1.8) MB, 0.71 / 0.00 ms  (average mu = 0.017, current mu = 0.007) last resort; GC in old space requested


<--- JS stacktrace --->


-
- Fatal JavaScript out of memory: CALL_AND_RETRY_LAST
-

/usr/bin/code: line 62: 19188 Trace/breakpoint trap   (core dumped) ELECTRON_RUN_AS_NODE=1 "$ELECTRON" "$CLI" --ms-enable-electron-run-as-node "$-"
```
Basically when I try to open VS Code in Fedora 39, sometimes (not always) it will throw a 133 error.
![Screenshot_20231227_222714](https://github-com/microsoft/vscode/assets/22721781/990e92e5-077a-403d-b7c7-4fdc475ea81f)
